### ISSUE TITLE
[Accessibility]: Text contrast of 1.96 with its background is less than the WCAG AA minimum requirements for text of size 12px and weight of 400

### ISSUE THREAD

**Issue by yashgarg1719** [carbon-design-system/carbon-charts#1406]
### Contact Details

yashgarg@kyndryl.com

### Environment

Browser, Automated testing tool and ruleset, Assistive technology used to verify

### Issue
Text contrast of 1.96 with its background is less than the WCAG AA minimum requirements for text of size 12px and weight of 400.

### Expectation
When text and its background colors have less than a 4.5 to 1 contrast ratio it can be difficult for people with moderately low vision to read the text without a contrast-enhancing technology. 

For larger text of 18 point or more, or bold 14 point text, the text and background colors must give at least a 3 to 1 contrast ratio.

For large text that is at least 18 point or at least 14 point when bold, adjust the color of the text or its background to give at least a 3 to 1 contrast ratio; Or, for body text that is less than 18 point or less than 14 point when bold, adjust the color of the text or its background to give at least a 4.5 to 1 contrast ratio.

The contrast ratio of text with its background must meet WCAG 2.1 AA requirements

Rule ID: IBMA_Color_Contrast_WCAG2AA Reason

### Testing environment details

- Operating System: MacOS
- Browser: Firefox 101.0(64-bit)

### Screenshot

![image](https://user-images.githubusercontent.com/108860673/177718949-48461656-ad66-41f4-a080-83d70fbfd0af.png)

### What WCAG 2.1 checkpoint does the issue violate?

IBM 1.4.3 Contrast (Minimum) WCAG technique G18 WCAG technique G145

### Version

`@carbon/charts@0.41.79`

**Comment by theiliad** [comment]
Going to close this out, I'm not able to see this violation using the scanner anymore.
